src/components/checkboxCollapsible/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { FC, Fragment, useState } from 'react';
+
+import { Box, Collapse } from '@chakra-ui/react';
+
+import Stack from '../stack';
+import { ChevronDownLargeIcon } from '../icons';
+import Flex from '../flex';
+import Divider from '../divider';
+import Checkbox, { CheckboxProps } from '../checkbox';
+import CollapsibleCheckbox from './CollapsibleCheckbox';
+
+interface CollapsibleCheckboxGroupProps extends CheckboxProps {
+  checkboxes?: CheckboxProps[];
+  addDividerAfterIndex?: number[];
+  size?: 'sm' | 'lg';
+}
+
+const CollapsibleCheckboxGroup: FC<CollapsibleCheckboxGroupProps> = ({
+  name,
+  label,
+  value,
+  onChange,
+  isChecked,
+  isDisabled,
+  isInvalid,
+  size = 'lg',
+  isIndeterminate,
+  addDividerAfterIndex,
+  variant = 'alignCenter',
+  checkboxes,
+}) => {
+  const [isOpen, setIsOpen] = useState(isIndeterminate || isChecked);
+  return (
+    <Stack spacing="lg">
+      <Flex alignItems="center">
+        <CollapsibleCheckbox
+          name={name}
+          icon={
+            <ChevronDownLargeIcon
```

Comment:
this should be a button. Consider using an Icon button and also pass an aria-label to it since there is no visible label